packages/utils/test/ip-port-to-multiaddr.spec.ts

```diff
@@ -19,7 +19,7 @@ describe('IP and port to Multiaddr', () => {
   it('creates multiaddr from valid IPv4 in IPv6 IP and port', () => {
     const ip = '0:0:0:0:0:0:101.45.75.219'
     const port = '9090'
-    expect(ipPortToMultiaddr(ip, port).toString()).to.equal(`/ip4/101.45.75.219/tcp/${port}`)
+    expect(ipPortToMultiaddr(ip, port).toString()).to.equal(`/ip6/::652d:4bdb/tcp/${port}`)
```

Comment:
`@chainsafe/is-ip` doesn't support detecting [IPv4 in IPv6 addresses](https://www.ibm.com/docs/en/ts4500-tape-library?topic=functionality-ipv4-ipv6-address-formats) so we lose support for returning an `IPv4` address in this case.

Node.js recognises these sorts of addresses as `IPv6` so I don't know if this is a problem or not.

E.g.:


```console
% node
Welcome to Node.js v18.14.0.
Type ".help" for more information.
> const net = require('net')
undefined
> net.isIP('0:0:0:0:0:0:101.45.75.219')
6
```

Reply:
We should be able to detect dual ipv6 addresses by simply testing for octects (or their delimiter, a period)

Reply:
I need to test the browser usecase. if this is showing as ipv6, we should be able to just wrap in brackets for URL construction.

Reply:
```bash
> new URL('http://[::101.45.75.219]')
URL {
  href: 'http://[::652d:4bdb]/',
  origin: 'http://[::652d:4bdb]',
  protocol: 'http:',
  username: '',
  password: '',
  host: '[::652d:4bdb]',
  hostname: '[::652d:4bdb]',
  port: '',
  pathname: '/',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
```